modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ExportObjectFolderMVCResourceCommand.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.object.definitions.portlet.action;
+
+import com.liferay.object.admin.rest.dto.v1_0.ObjectFolder;
+import com.liferay.object.admin.rest.resource.v1_0.ObjectFolderResource;
+import com.liferay.object.constants.ObjectPortletKeys;
+import com.liferay.object.web.internal.util.JSONObjectSanitizerUtil;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.portlet.PortletResponseUtil;
+import com.liferay.portal.kernel.portlet.bridges.mvc.BaseMVCResourceCommand;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCResourceCommand;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.ContentTypes;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.portal.kernel.util.Time;
+import com.liferay.portal.kernel.util.WebKeys;
+
+import javax.portlet.ResourceRequest;
+import javax.portlet.ResourceResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Marco Leo
+ */
+@Component(
+	property = {
+		"javax.portlet.name=" + ObjectPortletKeys.OBJECT_FOLDERS,
+		"mvc.command.name=/object_folders/export_object_folder"
+	},
+	service = MVCResourceCommand.class
+)
+public class ExportObjectFolderMVCResourceCommand
+	extends BaseMVCResourceCommand {
+
+	@Override
+	protected void doServeResource(
+			ResourceRequest resourceRequest, ResourceResponse resourceResponse)
+		throws Exception {
+
+		ObjectFolderResource.Builder builder =
+			_objectFolderResourceFactory.create();
+
+		ThemeDisplay themeDisplay = (ThemeDisplay)resourceRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+
+		ObjectFolderResource objectFolderResource = builder.user(
+			themeDisplay.getUser()
+		).build();
+
+		long objectFolderId = ParamUtil.getLong(
+			resourceRequest, "objectFolderId");
+
+		ObjectFolder objectFolder = objectFolderResource.getObjectFolder(
+			objectFolderId);
+
+		JSONObject objectFolderJSONObject = _jsonFactory.createJSONObject(
```

Comment:
Could you check if we need [these](https://github.com/liferay-objects/liferay-portal/blob/66aa670df749419197a5e745724fff521c225834/modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ExportObjectDefinitionMVCResourceCommand.java#L75-L97) steps of the `ExportObjectDefinitionMVCResourceCommand`?

Reply:
Thanks, i had a little misconception about them. Fixed!